src/analysis/complex/isometry.lean

```diff
@@ -49,6 +49,17 @@ def rotation : circle →* (ℂ ≃ₗᵢ[ℝ] ℂ) :=
 
 @[simp] lemma rotation_apply (a : circle) (z : ℂ) : rotation a z = a * z := rfl
 
+/-- Rotations as continuous `ℂ`-linear maps-/
+def rotation_clm (a : circle) : ℂ →L[ℂ] ℂ :=
```

Comment:
Does this need to exist?  Can you just use `(rotation a).to_continuous_linear_equiv`?

Reply:
I have to use that because `rotation a` is not defined to be a complex-linear map. So the `to_continuous_linear_map` won't give me the desired linearity.

Reply:
I see, but can't you just use `a • continuous_linear_map.id ℂ ℂ` without a separate definition?